src/develop/components/SearchDoc.js

```diff
@@ -70,17 +86,20 @@ var SearchDoc = React.createClass({
             <dt><code>inline        true|false</code></dt>
             <dd>Indicates that the search input should always
               be visible.</dd>
-            <dt><code>onChange      {"function ({text}) {...}"}</code></dt>
-            <dd>Function that will be called when the user types some text.</dd>
+            <dt><code>onChange      {"function ({value: , label: }|{string}, selected) {...}"}</code></dt>
```

Comment:
I missed that we added a second argument to onChange. Can we revert to the original onChange and rely on the deprecation guidance to get users to change?